test/strings/basic.jl

```diff
@@ -556,8 +556,11 @@ end
 end
 
 @testset "prevind and nextind" begin
-    let strs = Any["∀α>β:α+1>β", GenericString("∀α>β:α+1>β")]
-        for i in 1:2
+    let str = "∀α>β:α+1>β", strs = Any[str, GenericString(str),
+                                        SubString(str, 1), SubString(str, 1, 4),
```

Comment:
Incorrect indentation.

Reply:
those multibyte characters - on my terminal it looked correct :(.